docs/docsite/rst/user_guide/playbooks_blocks.rst

```diff
@@ -43,7 +43,9 @@ All tasks in a block inherit directives applied at the block level. Most of what
 
 In the example above, the 'when' condition will be evaluated before Ansible runs each of the three tasks in the block. All three tasks also inherit the privilege escalation directives, running as the root user. Finally, ``ignore_errors: yes`` ensures that Ansible continues to execute the playbook even if some of the tasks fail.
 
-Names for tasks within blocks have been available since Ansible 2.3. We recommend using names in all tasks, within blocks or elsewhere, for better visibility into the tasks being executed when you run the playbook.
+.. versionadded:: 2.3
+
+    The ``name:`` keyword for ``block:`` was added in Ansible 2.3.
```

Comment:
```suggestion
Names for blocks themselves and for tasks within blocks have been available since Ansible 2.3. We recommend using names in all tasks, within blocks or elsewhere, for better visibility into the tasks being executed when you run the playbook.
```

Reply:
Hey @acozine. Thanks for the quick review.

As for the suggested rewording. 
It says `Names for blocks themselves and for tasks within blocks have been available since 2.3`. 

Whenever I try to run a block that has tasks with names with an Ansible version prior 2.3 it seems to run fine, which is why this seems confusing to me, as I would expect it to fail.
For me it is only when I add a name to the block itself that it errors.

I understand that we are talking about very old versions here, I would just like to avoid publishing wrong information.

Thanks

Reply:
@FrontSide I revised the suggestion.